CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Added
+-  New functionality `to_json()` to support outputing instances to files
+
```

Comment:
We already have an "Added" section below.

Reply:
Oh, now I see the structure of the file(using vim). Previously, I thought I need to have "###Added" for every feature. Thanks for pointing it out!